packages/amplify-appsync-simulator/src/server/subscription/websocket-server/utils/decode-header.ts

```diff
@@ -0,0 +1,10 @@
+import { parse } from 'url';
+export function decodeHeaderFromQueryParam(rawUrl: string, paramName: string = 'header'): Record<string, any> {
+  const url = parse(rawUrl);
+  const params = new URLSearchParams(url.query);
```

Comment:
Don't we need to url decode querystring parameters in case they're coming in escaped form?